wrappers/core18.go

```diff
@@ -69,6 +69,7 @@ func writeSnapdToolingMountUnit(sysd systemd.Systemd, prefix string, opts *AddSn
 	content := []byte(fmt.Sprintf(`[Unit]
 Description=Make the snapd snap tooling available for the system
 Before=snapd.service
+Before=systemd-udevd.service
```

Comment:
systemd-udevd.service is before sysinit.target, so we would also need DefaultDependencies=no